content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -144,3 +146,15 @@ the CEL expressions that failed, respectively.
 The `validationActions` shows what actions were taken for this validation failure.
 See [Validating Admission Policy](/docs/reference/access-authn-authz/validating-admission-policy/)
 for more details about `validationActions`.
+
+## Event User.Extra Fields
+
+### authentication.kubernetes.io/credential-id
+
+This field contains the credential ID that was used to authenticate this request.
+
+For X.509 certificates, this is the SHA-256 fingerprint of the certificate, in the form `X509SHA256=<hexadecimal-encoded-hash>`.
```

Comment:
We typically use `:` to separate key from value (assuming YAML or pseudo-YAML). We can use JSON instead but then this should be a valid fragment of a JSON document, _and_ the rest of the page would need updating to match.

Watch out for the `=` here; it doesn't look like JSON or YAML.

Reply:
The User.Extra field is map[string][]string, in this case the map entry looks like:

```
Extra: map[string][]string {
  "authentication.kubernetes.io/credential-id": []string{"X509SHA256=abcdefghijkl..."},
}
```

The prefix and `=` is part of the value.

Reply:
Ah, OK. I recommend adding an example audit record that has this field set.